rskj-core/src/main/java/org/ethereum/db/DatabaseImpl.java

```diff
@@ -96,14 +95,10 @@ public void close() {
         keyValueDataSource.close();
     }
 
-    public List<ByteArrayWrapper> dumpKeys() {
-
-        ArrayList<ByteArrayWrapper> keys = new ArrayList<>();
-
-        for (byte[] key : keyValueDataSource.keys()) {
-            keys.add(ByteUtil.wrap(key));
-        }
-        Collections.sort(keys);
-        return keys;
+    public <T> List<T> dumpKeys(Function<byte[], T> mapper) {
+        return keyValueDataSource.keys()
+                .stream()
+                .map(mapper)
+                .collect(Collectors.toList());
```

Comment:
don't we need to sort them anymore?

Reply:
Great catch! I checked this method and the only usage is in `DetailsDataStore`, where the keys are immediately added to a `HashSet`. And the `HashSet` has no order, so the `sorted` call was useless.